README.md

```diff
@@ -4,7 +4,49 @@ Exercism Exercises in OCaml
 
 ## Contributing Guide
 
-Please see the [contributing guide](https://github.com/exercism/x-api/blob/master/CONTRIBUTING.md#the-exercise-data)
+Contributions to the OCaml track (or any other part of Exercism) are very welcome!
+
+Please see the [contributing guide](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md) for overall guidance. The below
+notes contain a few details specific to Ocaml.
+
+## Prerequisites
+
+The OCaml track assumes installation of OCaml version 4.03.0, and installation of Core, OUnit, and React (for the Hangman exercise).
+Assuming you have opam, these can be installed with
+```bash
+opam install core ounit react
+```
+
+## Running Tests
+
+To run all the tests, type `make` from the top level xocaml directory.
+
+To run tests for an individual exercise, `make test-assignment ASSIGNMENT=luhn`
+
+## Adding an Exercise
+
+The [contributing guide](https://github.com/exercism/x-common/blob/master/CONTRIBUTING.md) provides guidance on
+how to add a new exercise, or port an existing exercise from another language track.
+
+A sample exercise folder layout is below. Makefile and .merlin can be copied from any of the other exercises.
+
+```
+└── exercises
+    └── hello-world
```

Comment:
Maybe it is better to name this `sample`, or `SAMPLE` and mention that it should be named according to the exercise slug.